### PR TITLE
Adds tests for PWA stores

### PR DESCRIPTION
**Pull request by orballo**
fixes https://github.com/wp-pwa/saturn-theme/issues/312

### CODE REVIEW

File: src/pwa/stores/__tests__/client.tests.js [hunk 1]
```diff
@@ -0,0 +1,553 @@
+import * as mobx from 'mobx';
+import * as mst from 'mobx-state-tree';
+import * as utils from '../utils';
+import * as scrollMiddleware from '../middleware/scroll';
+import * as progressMiddleware from '../middleware/progress';
+import client from '../client';
+
+describe('Theme › PWA › Stores › Client', () => {
+  test('`requestFirstExtracted()` should request lists if there is any marked as horizontal extract', async () => {
```
Comment (luisherranz) [inline src/pwa/stores/__tests__/client.tests.js]: You can delete `async` here.

Comment (orballo) [reply src/pwa/stores/__tests__/client.tests.js]: Done. Thanks!

**Comment by orballo** [comment]
Yes, you are right. They should be in `connection`.

**Pull request merged**